src/lib/MagicSpoon.js

```diff
@@ -10,6 +10,32 @@ import directory from '../directory';
 const MagicSpoon = {
   async Account(Server, keypair, onUpdate) {
     let sdkAccount = await Server.loadAccount(keypair.publicKey())
+
+    let effectsHistory = await Server.effects().forAccount((keypair.publicKey())).limit(200).order('desc').call();
```

Comment:
This seems quite wasteful to make up to 201 requests per account. Perhaps we can just load tx details when the user hits the transaction history page and then cache the results.